packages/volto/locales/volto.pot

```diff
@@ -5035,6 +5069,11 @@ msgstr ""
 msgid "rrule_years"
 msgstr ""
 
+#. Default: "save"
```

Comment:
This is redundant to the capitalized `Save` above. Please remove this translation block and refactor the instances of `save` in your code to `Save` so that they use that translation. You'll also need to add the component to the above `Save` translation block.